docs/source/installation/methods.rst

```diff
@@ -8,3 +8,5 @@ ceph-ansible can deploy Ceph either in a non-containerized context (via packages
 
    non-containerized
    containerized
+
+The difference here is that you don't have the rbd command on the host when using the containerized deployment so everything related to ceph needs to be executed within a container. So in the case there is software like e.g. Open Nebula which requires that the rbd command is accessible directly on the host (non-containerized) then you need to have to install the rbd command by yourself on those servers outside of containers. (Or make sure that this software somehow runs within containers as well and that it can access rbd.
```

Comment:
```suggestion
The difference here is that you don't have the rbd command on the host when using the containerized deployment so everything related to ceph needs to be executed within a container. So in the case there is software like e.g. Open Nebula which requires that the rbd command is accessible directly on the host (non-containerized) then you have to install the rbd command by yourself on those servers outside of containers. (Or make sure that this software somehow runs within containers as well and that it can access rbd.
```

Reply:
@modir